packages/number-line/src/number-line/graph/tick-utils.js

```diff
@@ -121,43 +133,45 @@ export const simpleRange = (start, end, interval) => {
   interval = math.fraction(interval);
 
   const positiveRange = math.larger(end, 0) ? zeroBasedRange(fmax(0, start), end, interval) : [];
+
   const negativeRange = math.smaller(start, 0) ? zeroBasedRange(fmin(0, end), start, math.multiply(interval, -1)) : [];
+  let together = negativeRange.concat(positiveRange);
 
-  return uniqWith([...negativeRange, ...positiveRange], math.equal);
+  const out = uniqWith(together, math.equal);
+  return out;
 };
 
 export const closeTo = (a, b, precision) => {
   precision = precision || 2;
   const expectedDiff = Math.pow(10, -precision) / 2;
   const receivedDiff = Math.abs(a - b);
-
   return receivedDiff < expectedDiff;
 };
 
-const limit = (value, min, max) => {
-  if (math.smaller(value, min)) {
+const limit = (v, min, max) => {
+  if (math.smaller(v, min)) {
     return min;
   }
 
-  if (math.larger(value, max)) {
+  if (math.larger(v, max)) {
     return max;
   }
 
-  return value;
+  return v;
 };
 
-export const getMinorLimits = (domain) => {
+export const minorLimits = (domain, width) => {
```

Comment:
`minorLimits` was changed to `getMinorLimits` recently. Please keep the `getMinorLimimits` as it is on develop branch

Reply:
@andreeimiron As per change list minor limits needs to be calculated as follow The current logic in develop branch does not follow this requirements that's why I modified it. Please let me know.
![image](https://github.com/pie-framework/pie-elements/assets/118424906/40d7f583-198e-4c13-883a-6ed05466a453)